plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeHistoryTable.java

```diff
@@ -92,6 +93,7 @@ protected List<Page> buildPages(ConnectorSession session, PageListBuilder pagesB
             pagesBuilder.beginRow();
 
             pagesBuilder.appendBigint(commitInfoEntry.version());
+            commitInfoEntry.inCommitTimestamp().ifPresentOrElse(inCommitTimestamp -> pagesBuilder.appendTimestampTzMillis(inCommitTimestamp, timeZoneKey), pagesBuilder::appendNull);
```

Comment:
Why are we presenting the timestamp in a timezone derived from user session ?
The spec says `represents the time (in milliseconds since the Unix epoch)`, so we should use UTC timezone here